controller/models/Instance.go

```diff
@@ -13,7 +13,83 @@
 // limitations under the License.
 package models
 
+import (
+	"bytes"
+	"encoding/json"
+)
+
 // Instance is the external view of instances
-type Instance struct {
+type InstanceView struct {
 	Address string `json:"address"`
 }
+
+// InstanceState define instance state
+type InstanceState int
+
+const (
+	// Active indicates an instance is healthy
+	Active InstanceState = iota
```

Comment:
I defined it sort of arbitrarily. now that we have m3db integration, it may make sense to change the states to something in line with what topology defines.

Reply:
topology does not have state for host, actually i am not really seeing the state being actually used anywhere. maybe i will just keep the instance only to simplify since we will deprecate MembershipMutator eventually after migration.

Reply:
no, it was not used. if the status will not be on hosts, we can remove it.